packages/manager/modules/nutanix/src/dashboard/nodes/node/template.html

```diff
@@ -0,0 +1,52 @@
+<oui-header
+    class="mb-3"
+    heading="{{:: $ctrl.node.description }}"
+    description="{{:: $ctrl.nodeId }}"
+>
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            state="nutanix.dashboard.nodes.node.general-info"
+            state-params="{
+                serviceName: $ctrl.serviceName,
+                nodeId: $ctrl.nodeId
+            }"
+            ><span data-translate="nutanix_node_general_info"></span>
+        </oui-header-tabs-item>
+        <oui-header-tabs-item
+            state="nutanix.dashboard.nodes.node.interventions"
+            state-params="{
+                serviceName: $ctrl.serviceName,
+                nodeId: $ctrl.nodeId
+            }"
+            ><span data-translate="nutanix_node_interventions"></span>
+        </oui-header-tabs-item>
+        <oui-header-tabs-item
+            state="nutanix.dashboard.nodes.node.ipmi"
+            state-params="{
+                serviceName: $ctrl.serviceName,
+                nodeId: $ctrl.nodeId
+            }"
+            ><span>IPMI</span>
+        </oui-header-tabs-item>
+        <oui-header-tabs-item
+            state="nutanix.dashboard.nodes.node.tasks"
+            state-params="{
+                serviceName: $ctrl.serviceName,
+                nodeId: $ctrl.nodeId
+            }"
+            ><span data-translate="nutanix_node_tasks"></span>
+        </oui-header-tabs-item>
+    </oui-header-tabs>
+    <oui-guide-menu
+        data-text="{{:: 'nutanix_node_guides_header' | translate }}"
+    >
+        <oui-guide-menu-item
+            href="{{ $ctrl.dedicatedServerGuideUrl }}"
+            data-external
+        >
+            <span data-translate="nutanix_node_guides_header_all_guides"></span>
+        </oui-guide-menu-item>
+    </oui-guide-menu>
+</oui-header>
+<div class="alert-server mt-3 mb-3" data-ovh-alert="nutanix_node_alert"></div>
```

Comment:
Is there any chance to remove this class `alert-server`?


```suggestion
<div class="alert-server my-3" data-ovh-alert="nutanix_node_alert"></div>
```